update cert-manager CRD version to v1 (#1582)
pingcap/docs-tidb-operator: en/enable-tls-between-components.md

    > The Secret objects you created must follow the above naming convention. Otherwise, the deployment of the TiDB components will fail.

2. Deploy the cluster, and set `.spec.tlsCluster.enabled` to `true`.

    > **Note:**
    >

>     > This field can't be modified after the cluster is created, otherwise the cluster upgrade will fail.

Reviewer: ```suggestion
    > After the cluster is created, do not modify this field; otherwise, the cluster will fail to upgrade.
```
